Pipfile

```diff
@@ -5,13 +5,18 @@ name = "pypi"
 
 [packages]
 python-dotenv = "~=1.0"
-requests = "~=2.26"
 ruff = "~=0.7"
 tqdm = "~=4.66"
+requests = "*"
+wget = "*"
 
 [dev-packages]
 pytest = "~=8.3"
 pytest-cov = "~=5.0"
 
 [requires]
 python_version = "3.12"
+
+[scripts]
+    download_data = "python download_data.py"
```

Comment:
These need to be `pipenv run python -m mediabridge.download_data`. The `pipenv run` is needed to make the scripts run in the pipenv environment.